src/masonite/orm/blueprint/Blueprint.py

```diff
@@ -113,6 +122,19 @@ def datetime(self, column, nullable=False):
         self._columns += (self._last_column,)
         return self
 
+    def timestamp(self, column, nullable=False):
+        self._last_column = self.new_column("timestamp", column, None, nullable)
+        self._columns += (self._last_column,)
+        return self
+
+    def timestamps(self):
+        created_at = self.new_column('timestamp', 'created_at', None, nullable=False).use_current()
+        updated_at = self.new_column('timestamp', 'updated_at', None, nullable=False).use_current()
```

Comment:
Should we add some of these options here in the parameters list? like 

```python
def timestamps(self, nullale=False, current=True):
    #..
```

This way we can also easily add the options as parameters. I think adding the ability to do that is simple enough